made-with.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default.html
+title: Made With Servo
+---
+
+<style type="text/css">
+
+.made-with-container {
+  gap: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 32px;
+/*  align-items: stretch;*/
+}
+
+.made-with-item {
+  display: block;
+  width: 400px;
+  flex: 0 0 400px;
+  border: 2px solid #333;
+  cursor: pointer;
+  color: inherit;
+  text-decoration: inherit;
+}
+
+.made-with-item:hover {
+  background-color: rgba(27, 131, 129, 0.5);
+  background-color: #EAEAEA;
+}
+
+.made-with-text {
+  padding: 20px;
+}
+
+.made-with-text > p {
+  padding: 0;
+}
+
+</style>
+
+<div class="inner-container">
+
+<h1>{{ title }}</h1>
+<p class="subtitle" style="margin-bottom: 32px">
+  This page showcases projects that are built with Servo or Servo components
+</p>
+
+{% for group in madewith %}
+
+### {{group.name}}
+
+<div class="made-with-container">
+
+  {% for project in group.projects %}
+
+  <a class="made-with-item" href="{{project.link}}" target="_blank">
+    <img src="{{ project.img | url }}" />
```

Comment:
It'd be nice to add some `alt` text to images, maybe it could be part of the `.json` data.

Reply:
What do you reckon someone using a screenreader would find a useful description here? In my mind the surrounding text already describes the product and the images don't actually convey much useful information / are mostly decorative. Perhaps `alt=" "` to get screenreaders to skip the image entirely is appropriate? (https://css-tricks.com/alt-text-not-always-needed/)